cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,37 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const user = {
+    username: 'romda.com',
+    password: '123Qwert!'
+  };
 
+  const book = {
+    descreption: 'Like it or not, JavaScript is everywhere'
+  };
+  before(() => {
+    cy.visit('/');
   });
 
-  it('', () => {
-
+  it('user should be able to login and search book and delete it', () => {
+    cy.findByPlaceholder('UserName').type(user.username);
+    cy.findByPlaceholder('Password').type(user.password);
+    cy.get('#login').click();
+    cy.get('#userName-value').should('contain', user.username);
+    cy.url().should('include', '/profile');
+    cy.get('#gotoStore').click();
+    cy.get('#searchBox').type('Speaking JavaScript');
+    cy.get('a[href="/books?book=9781449365035"]').click();
+    cy.get('#description-wrapper').should('contain', book.descreption);
+    cy.get('.text-right > #addNewRecordButton').click();
```

Comment:
better to user here `#addNewRecordButton` selector only

Reply:
fixed selector